src/doc/trpl/looping.md

```diff
@@ -18,7 +18,7 @@ for (x = 0; x < 10; x++) {
 Instead, it looks like this:
 
 ```{rust}
-for x in range(0, 10) {
+for x in (0..10) {
```

Comment:
Nice I did not know that.
However not sure about removing the parens mainly as it allows for easier edits.
The second example
```{rust}
for x in (0..10) {
    if x % 2 == 0 { continue; }
    println!("{}", x);
}
```
could be refactored to 
```{rust}
for x in (0..10).filter(|&x| (x % 2 != 0)) {
    println!("{}", x);
}
```
where parens become mandatory right?
Keeping the parens saves you from jumping to the begging to add them later. 

However if there is a style guide/convention I am breaking let me know and I will make the edit